pages/SeasonOfDocs-2019.md

```diff
@@ -0,0 +1,68 @@
+# Project Ideas
+
```

Comment:
The content is great but for this to be served properly on hydraecosystem.org, we need to add this section at the top of the page.
```
---
layout: page
title: Google Season of Docs 2019 | Hydraecosystem.org
permalink: /gsod-2019
---

Reply:
Ahh yes..done :)

Reply:
It's just my opinion but Amruta don't you think we should keep the URL short? `gsod-19` seems like a better choice to me.

Reply:
Yes, I would have preferred that. But the organizers announced yesterday that they don't want to abbreviate the name (to avoid some cultural implications). Hence the long-form.

Reply:
it short enough, can be good to have the full name for search indexing reasons